src/components/Modal.jsx

```diff
@@ -1,54 +1,66 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Photo from './Photo.jsx';
+import Photo from './Photo';
```

Comment:
I wasn't able to import things when I removed the .jsx in order to comply with eslint. Did the same happen to you? How'd you fix it?

Reply:
Same.

Reply:
git mv flie.jsx file.js
That renames files (in a way git can track.)